base/bitarray.jl

```diff
@@ -1272,29 +1272,45 @@ end
 (>>)(B::BitVector, i::Int32) = B >>> i
 (>>)(B::BitVector, i::Integer) = B >>> i
 
+function rol!(dest::BitVector, src::BitVector, i::Integer)
+    if length(dest) == length(src)
+        n = length(dest)
+        i %= n
+        i == 0 && return dest
```

Comment:
Would it not be better if we returned something like ror!(dest, src, -i) for  
`i < 0`  which would look into the other cases ??